Simplenote/src/main/java/com/automattic/simplenote/LastSyncTimeCache.java

```diff
@@ -1,23 +1,33 @@
 package com.automattic.simplenote;
 
+import android.content.Context;
+import android.content.SharedPreferences;
 import android.util.Log;
 
+import androidx.preference.PreferenceManager;
+
 import com.automattic.simplenote.models.Note;
 import com.simperium.client.Bucket;
 
 import java.util.Calendar;
-import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Set;
 
 public class LastSyncTimeCache {
-    private HashMap<String, Calendar> mSyncTimes = new HashMap<>();
-    private HashSet<String> mUnsyncedKeys = new HashSet<>();
-    private Set<SyncTimeListener> mListeners = new HashSet<>();
+    public static final long DEFAULT_LAST_SYNC_TIME = -1;
+
     private static final String TAG = LastSyncTimeCache.class.getSimpleName();
 
-    public Calendar getLastSyncTime(String key) {
-        return mSyncTimes.get(key);
+    private final HashSet<String> mUnsyncedKeys = new HashSet<>();
+    private final Set<SyncTimeListener> mListeners = new HashSet<>();
+    private final SharedPreferences mPreferences;
+
+    public LastSyncTimeCache(Context context) {
+        mPreferences = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
WDYT about moving those to a dedicated preferences file, instead of the default one? IMO think it will be cleaner, and keep using the default preferences file for "user preferences"